test/integration/tests.js

```diff
@@ -3,19 +3,26 @@ QUnit.test('agree to terms', function (assert) {
 
   // Select the mock app root
   var app = $('iframe').contents().find('#app-content .mock-app-root')
+  // var button = $('.agree')
+  // button.removeAttr('disabled')
 
-  // Agree to terms
-  app.find('button').click()
+  app.find('.markdown').prop('scrollTop', 100000000)
 
-  // Wait for view to transition:
+
+  // Agree to terms
   wait().then(function() {
+    app.find('button').click()
+    wait().then(function() {
```

Comment:
I tried doing it the proper way, but for some reason, it keeps failing.

```
  wait().then(function() {
    app.find('button').click()
  }).then(function() {
    wait()
  }).then(function() {
    var title = app.find('h1').text()
    assert.equal(title, 'MetaMask', 'title screen')

    var buttons = app.find('button')
    assert.equal(buttons.length, 2, 'two buttons: create and restore')

    done()
  })
```

and the tests:

```
not ok 1 Chrome 53.0 - agree to terms
    ---
        actual: >
            MetaMaskTerms of Use
        expected: >
            MetaMask
        stack: >
                at http://localhost:7357/3389/test/integration/tests.js:31:12
        message: >
            title screen
        Log: |
    ...
not ok 2 Firefox 49.0 - agree to terms
    ---
        actual: >
            MetaMaskTerms of Use
        expected: >
            MetaMask
        stack: >
            @http://localhost:7357/3777/test/integration/tests.js:31:5
            
        message: >
            title screen
        Log: |
    ...

1..2
# tests 2
# pass  0
# skip  0
# fail  2
```